docker/Dockerfile

```diff
@@ -30,6 +30,11 @@ RUN	GRPC_HEALTH_PROBE_VERSION=v0.3.6 \
 	&& chmod +x /usr/bin/grpc_health_probe \
 	&& ldconfig /usr/local/lib
 
+RUN FFMPEG_SHA=b76053d8bf322b197a9d07bd27bbdad14fd5bc15 git clone --depth 1 https://git.ffmpeg.org/ffmpeg.git /ffmpeg \
```

Comment:
I thought we already had local ffmpeg for rtmp input and output 😕 
https://github.com/livepeer/go-livepeer/blob/master/media/rtmp2segment.go#L59

Reply:
Oh I see now, you've removed the `apt install` and are compiling it instead